tools/CMakeLists.txt

```diff
@@ -249,26 +226,6 @@ iree_cc_binary(
   INSTALL_COMPONENT IREETools-Runtime
 )
 
-iree_cc_binary(
-  NAME
-    iree-run-trace
```

Comment:
Found another reference with a project-wide search for `iree-run-trace`. The diagrams in https://github.com/openxla/iree/tree/main/docs/website/docs/assets/images , which are included on the website and in the repo README:

![image](https://github.com/openxla/iree/assets/4010439/d0fc3224-808a-4367-9c26-8133de3eb325)

Should update the list there. I remember the light mode and dark mode exports from the excalidraw file being a bit different in order to have the right background colors for all items. Find to leave that to a follow-up, and I can help with it if you'd like.